scripts/e2e-installs.test.js

```diff
@@ -27,31 +34,33 @@ describe('Installation', () => {
   });
 
   describe('Dependencies', () => {
-    describe('Node', () => {
-      test('get installed by default', () => {
-        execSync(
-          `yarn start ${appPath} \
-            --template "InstantSearch.js"`,
-          { stdio: 'ignore' }
-        );
-
-        expect(fs.lstatSync(`${appPath}/node_modules`).isDirectory()).toBe(
-          true
-        );
-      });
-    });
-
-    describe('CocoaPods', () => {
-      test('get installed by default', () => {
-        execSync(
-          `yarn start ${appPath} \
-            --template "InstantSearch iOS"`,
-          { stdio: 'ignore' }
-        );
-
-        expect(fs.lstatSync(`${appPath}/Pods`).isDirectory()).toBe(true);
+    templates
+      .filter(
+        templatePath =>
+          // The Android template relies on Android Studio and
+          // doesn't install dependencies
+          path.basename(templatePath) !== 'InstantSearch Android'
+      )
+      .forEach(templatePath => {
+        const templateName = path.basename(templatePath);
+
+        describe(templateName, () => {
+          test('get installed by default', () => {
+            execSync(
+              `yarn start ${appPath} \
+                --template "${templateName}"`,
+              { stdio: 'ignore' }
+            );
+
+            const dependenciesDirectory =
+              templateName === 'InstantSearch iOS' ? 'Pods' : 'node_modules';
```

Comment:
I would rather keep the tests a bit more simpler. If you want to tests all the templates installation, maybe scope them by categories (web, mobile)? But we could avoid to put too much logic in the test.

Reply:
I refactored it. Let me know if it's a bit better now.